docs/www/docs/contribute/99-troubleshooting.md

```diff
@@ -110,21 +115,24 @@ For eg:
 }
 ```
 
-**IMPORTANT: Make sure the `StaticSitesClient.json#checksum` and `StaticSitesClient.json#metadata.files.[OS].sha` values match!**
-6. For Linux and macOS, run `chmod +x /home/USER/.swa/deploy/1.0.020761/StaticSitesClient`
-7. Run `swa deploy --verbose silly` and make sure `SWA_CLI_DEPLOY_BINARY` is set correctly. If everything was configured correctly, the deploy should work.
-
+**IMPORTANT: Make sure the `StaticSitesClient.json#checksum` and `StaticSitesClient.json#metadata.files.[OS].sha` values match!** 6. For Linux and macOS, run `chmod +x /home/USER/.swa/deploy/1.0.020761/StaticSitesClient` 7. Run `swa deploy --verbose silly` and make sure `SWA_CLI_DEPLOY_BINARY` is set correctly. If everything was configured correctly, the deploy should work.
 
 How to compute SHA256 checksum:
-1. On Windows using Powershell: 
+
+1. On Windows using Powershell:
+
 ```
 PS C:\Users\USER> (Get-fileHash -Algorithm SHA256 .\.swa\deploy\VERSION\StaticSitesClient.exe).Hash.ToLower()
 ```
+
 2. On Linux:
+
 ```
 ➜ sha256sum ~/.swa/deploy/VERSION/StaticSitesClient | head -c 64
 ```
+
```

Comment:
Are these intentional changes or done as part of some formatting tool?

Reply:
not intentional Sulabh.

Reply:
Can we revert these changes and just keep one line change for the addition of the domain

Reply:
even if I revert these changes are being added automatically when a commit is made.

Reply:
All right. Then lets go ahead with the changes then